src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the beginning of the `Figure` class.